usaspending_api/transactions/delta_models/published_fabs.py

```diff
@@ -0,0 +1,120 @@
+PUBLISHED_FABS_COLUMNS = {
+    "published_fabs_id": {"delta": "INTEGER", "postgres": "UUID"},
+    "afa_generated_unique": {"delta": "STRING", "postgres": "TEXT"},
+    "action_date": {"delta": "STRING", "postgres": "TEXT"},
+    "action_type": {"delta": "STRING", "postgres": "TEXT"},
+    "action_type_description": {"delta": "STRING", "postgres": "TEXT"},
+    "assistance_type": {"delta": "STRING", "postgres": "TEXT"},
+    "assistance_type_desc": {"delta": "STRING", "postgres": "TEXT"},
+    "award_description": {"delta": "STRING", "postgres": "TEXT"},
+    "award_modification_amendme": {"delta": "STRING", "postgres": "TEXT"},
+    "awardee_or_recipient_legal": {"delta": "STRING", "postgres": "TEXT"},
+    "awardee_or_recipient_uniqu": {"delta": "STRING", "postgres": "TEXT"},
+    "awarding_agency_code": {"delta": "STRING", "postgres": "TEXT"},
+    "awarding_agency_name": {"delta": "STRING", "postgres": "TEXT"},
+    "awarding_office_code": {"delta": "STRING", "postgres": "TEXT"},
+    "awarding_office_name": {"delta": "STRING", "postgres": "TEXT"},
+    "awarding_sub_tier_agency_c": {"delta": "STRING", "postgres": "TEXT"},
+    "awarding_sub_tier_agency_n": {"delta": "STRING", "postgres": "TEXT"},
+    "business_categories": {"delta": "ARRAY<STRING>", "postgres": "ARRAY[]TEXT"},
+    "business_funds_ind_desc": {"delta": "STRING", "postgres": "TEXT"},
+    "business_funds_indicator": {"delta": "STRING", "postgres": "TEXT"},
+    "business_types": {"delta": "STRING", "postgres": "TEXT"},
+    "business_types_desc": {"delta": "STRING", "postgres": "TEXT"},
+    "cfda_number": {"delta": "STRING", "postgres": "TEXT"},
+    "cfda_title": {"delta": "STRING", "postgres": "TEXT"},
+    "correction_delete_ind_desc": {"delta": "STRING", "postgres": "TEXT"},
+    "correction_delete_indicatr": {"delta": "STRING", "postgres": "TEXT"},
+    "created_at": {"delta": "TIMESTAMP", "postgres": "TIMESTAMP"},
+    "face_value_loan_guarantee": {"delta": "NUMERIC(23,2)", "postgres": "NUMERIC"},
```

Comment:
On our other models, we have `NUMERIC(23,2)` for postgres on our `NUMERIC` types

Reply:
addressed